vendor.conf

```diff
@@ -52,3 +52,4 @@ gopkg.in/yaml.v2 4c78c975fe7c825c6d1466c42be594d1d6f3aba6
 github.com/tonistiigi/fsutil 0ac4c11b053b9c5c7c47558f81f96c7100ce50fb
 github.com/stevvooe/continuity cd7a8e21e2b6f84799f5dd4b65faf49c8d3ee02d
 golang.org/x/sync de49d9dcd27d4f764488181bea099dfe6179bcf0
+vbom.ml/util 928aaa586d7718c70f4090ddf83f2b34c16fdc8d
```

Comment:
can you include this change in the same commit that adds the vendored code?

given that this is a new vendor, it's probably ok as well to just squash your commits (for updating vendors we usually keep it in a separate commit, but perhaps it's ok to do everything in a single one to not complicate things)